.gitignore

```diff
@@ -1,7 +1,7 @@
 # Custom
 ########
 .vs/
-**/entity/market/Place
+**/entity/marketPlace
```

Comment:
Because this structure isn't standard, please move this part of the `.gitignore` file to the one in your Item's subdirectory.